evutil.c

```diff
@@ -160,6 +161,14 @@ evutil_open_closeonexec_(const char *pathname, int flags, unsigned mode)
 	return fd;
 }
 
+ev_off_t evutil_fd_filesize(evutil_socket_t fd)
+{
+    struct stat st;
```

Comment:
This file uses tabs not spaces

Reply:
b1f5c0491a6ad3a52c3309f5026d338d03d45e68